dpbench/console/run.py

```diff
@@ -58,6 +58,12 @@ def add_run_arguments(parser: argparse.ArgumentParser):
         default=False,
         help="Set if run polybench benchmarks.",
     )
+    parser.add_argument(
+        "--rodinia",
+        action=argparse.BooleanOptionalAction,
+        default=False,
```

Comment:
I guess, we can include rodinia benchmarks to be executed by default. @diptorupd , what do you think?

Reply:
I was thinking about it, and I guess it is better leave it disabled for now. In this case we need to update CI to test this code. You need to check it in two places: 
- conda-package.yml , use npbench as reference:
https://github.com/IntelPython/dpbench/blob/main/.github/workflows/conda-package.yml#L193-L196
- build-and-run.yml , you just need to run it along with all other benchmarks

Reply:
@ZzEeKkAa Sure we can do it as a later step, but we should enable the rodhinia benchmark running in GitHub CI at some point in near future.

Reply:
As discussed, I have added it as a part of the CI and is run using the build_and_run.yml inside the workflow.